### PR TITLE
Add a preloader to the website

### LINKED ISSUE #11: Add a Loader to the website
Make sure loader is animated, made by you and uses open code logo.

### PR DESCRIPTION
**Pull request by sggts04**
__11__ 

Fixes #11 

#### Short description of what this resolves:
Adds a preloader to the website.

#### Changes proposed in this pull request and/or Screenshots of changes:
Live Link: https://shreyasgupta.in/Opencode-Collaborative-20/

### CODE REVIEW

**Comment by Jigar3** [comment]
The preloader background color doesn't match with the overall color scheme of the website. Use some other color

**Comment by sggts04** [comment]
@Jigar3 done, I have made it match the website background, i.e. white.

**Comment by Jigar3** [comment]
Doesn't reflect in the live link

**Comment by sggts04** [comment]
@Jigar3 
https://shreyasgupta.in/Opencode-Collaborative-20/ 
Maybe Shift+F5 for hard refresh would work?
This is how its looking on my side
image

**Comment by Jigar3** [comment]
White looks to dull and also the preloader isn't that great in terms of look and feel. Try using some other preloader

**Comment by sggts04** [comment]
@Jigar3 new preloader up 
Shift + F5 the live link

**Comment by Jigar3** [comment]
still looks dull

**Comment by sggts04** [comment]
@Jigar3 now?

File: index.html [hunk 1]
```diff
@@ -344,7 +347,12 @@ <h1 id="participants">Participants and details</h1>
   <script type="text/javascript" src = "data/participants.json"></script>
 
 
-
+  <script>
+    $(window).load(function() {
+        // Animate loader off screen
+        $(".se-pre-con").delay(1000).fadeOut("slow");
```
Comment (Jigar3) [inline index.html]: why are you adding a delay. It is for something specific

**Review by Jigar3 — CHANGES_REQUESTED** [review]

**Comment by sggts04** [comment]
@Jigar3 removed the delay

**Comment by Jigar3** [comment]
squash your commits

**Comment by sggts04** [comment]
@Jigar3 done

**Comment by thesmallstar** [comment]
@sggts04 can you remove conflicts?

**Comment by sggts04** [comment]
@thesmallstar resolved

**Comment by sggts04** [comment]
@thesmallstar pls review

**Comment by NitinMadhukar** [comment]
@sggts04 squash commits

**Comment by sggts04** [comment]
@NitinMadhukar done

**Comment by sggts04** [comment]
@Jigar3 @thesmallstar please review

**Pull request merged**